frontend/wasm/src/code_translator/mod.rs

```diff
@@ -910,7 +910,6 @@ fn translate_br_table<B: ?Sized + Builder>(
 
         targets.push(depth);
     }
-    targets.sort();
```

Comment:
Let's add an assertion that the targets are in sorted order, because otherwise the binary search lowering used in the backend would produce incorrect code silently - I'd rather have it fail noisily if that implicit constraint is violated.